README.md

```diff
@@ -5,25 +5,50 @@ Create SaaS Boilerplate
 [![Downloads/week](https://img.shields.io/npm/dw/create-saas-boilerplate.svg?style=for-the-badge)](https://npmjs.org/package/create-saas-boilerplate)
 [![License](https://img.shields.io/npm/l/create-saas-boilerplate.svg?style=for-the-badge)](https://github.com/apptension/create-saas-boilerplate/blob/master/package.json)
 
+Create SaaS Boilerplate is a CLI tool designed to quickly set up a new local instance of the SaaS Boilerplate. It is a specialized tool that allows users to get started with the SaaS Boilerplate as quickly as possible. 
+
+
+---
+
+📖 **SaaS Boilerplate Documentation**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+
+🌟 **SaaS Boilerplate Page**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+
+---
+
+## Table of contents
 <!-- toc -->
 * [Usage](#usage)
 * [Commands](#commands)
 <!-- tocstop -->
-# Usage
+
+## Usage
 <!-- usage -->
+The usage for Create SaaS Boilerplate starts with installing the tool globally using npm.
```

Comment:
What happens when you run `oclif readme`? It won't override added descriptions?

Reply:
Auto generated description is ok. Maybe we could adjust description of the commands a bit but but for now I would keep the auto generated content as it is and just add the better description above